lib/bemhtml/entity.js

```diff
@@ -15,13 +15,13 @@ function Entity(bemxjst) {
   this.jsClass = null;
 
   // "Fast modes"
-  this.tag = new Match(this);
-  this.attrs = new Match(this);
-  this.mod = new Match(this);
-  this.js = new Match(this);
-  this.mix = new Match(this);
-  this.bem = new Match(this);
-  this.cls = new Match(this);
+  this.tag = new Match(this, 'tag');
+  this.attrs = new Match(this, 'attrs');
+  this.mod = new Match(this, 'mod');
```

Comment:
Didn't noticed before, what the purpose of this? There is no `mod` property in `bemjson` as well as `mod` with template body. We have `block('a').mod('m', 'v')(body)` but not `block('a').mod()(body)`.